website/docs/r/nsxt_edgegateway.html.markdown

```diff
@@ -0,0 +1,199 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_nsxt_edgegateway"
+sidebar_current: "docs-vcd-resource-nsxt-edge-gateway"
+description: |-
+  Provides a VMware Cloud Director NSX-T edge gateway. This can be used to create, update, and delete NSX-T edge gateways connected to external networks.
+---
+
+# vcd\_nsxt\_edgegateway
+
+Provides a VMware Cloud Director NSX-T edge gateway. This can be used to create, update, and delete NSX-T edge gateways connected
+to external networks.
+
+Supported in provider *v3.1+*
+
+~> **Note:** Only `System Administrator` can create an edge gateway.
```

Comment:
Just confirming, is this still true for NSX-T?

Reply:
Yes. And it makes sense because one needs to access external network to create it. And that would be a problem.

![image](https://user-images.githubusercontent.com/15804230/101899895-350c8900-3bb7-11eb-86c5-28975a8a0b41.png)